From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marc Bourgoin <themard85@gmail.com>
Date: Tue, 15 Oct 2024 22:19:31 -0600
Subject: [PATCH] touchscreen_mmi: Mark mode_type as maybe unused

../../../../../../kernel/motorola/sm7325/drivers/input/touchscreen/touchscreen_mmi/touchscreen_mmi_gesture.c:216:16: error: unused variable 'mode_type' [-Werror,-Wunused-variable]
        unsigned char mode_type = touch_cdev->gesture_mode_type;
                      ^
1 error generated.

Change-Id: Id01d5c2286156e6ee2dd1da2bd9b9a7c9bb883bf
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 .../input/touchscreen/touchscreen_mmi/touchscreen_mmi_gesture.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/touchscreen/touchscreen_mmi/touchscreen_mmi_gesture.c b/drivers/input/touchscreen/touchscreen_mmi/touchscreen_mmi_gesture.c
index 061dd86eff72..1d208b31d02b 100644
--- a/drivers/input/touchscreen/touchscreen_mmi/touchscreen_mmi_gesture.c
+++ b/drivers/input/touchscreen/touchscreen_mmi/touchscreen_mmi_gesture.c
@@ -213,7 +213,7 @@ static inline void update_poison_center(struct touch_event_data *tev)
 
 static void ts_mmi_single_tap_handler(struct ts_mmi_dev *touch_cdev)
 {
-	unsigned char mode_type = touch_cdev->gesture_mode_type;
+	unsigned char __maybe_unused mode_type = touch_cdev->gesture_mode_type;
 	ktime_t now, tmp;
 
 	if (!touch_cdev->single_tap_pressed) {
-- 
2.39.5

